TOC.md

```diff
@@ -10,9 +10,9 @@
   - [TiDB Limitations](/tidb-limitations.md)
   - [Credits](/credits.md)
   - [Roadmap](/tidb-roadmap.md)
-- Quick Start
-  - [Try Out TiDB](/quick-start-with-tidb.md)
-  - [Try Out HTAP](/quick-start-with-htap.md)
+- Get Started
+  - [Quick Start Using TiUP Playground](/quick-start-with-tidb.md)
+  - [Use TiFlash for HTAP](/quick-start-with-htap.md)
```

Comment:
@lilin90 Shall we also update the URL to match the title? Such as "use-tiflash-for-htap.md"

Reply:
Or we can just use `- [Quick Start with HTAP](/quick-start-with-htap.md)`